docs/config/README.md

```diff
@@ -142,26 +154,36 @@ To use an HD Wallet with Hardhat you should set your network's `accounts` field
 }
 ```
 
-## Solc configuration
+## Solidity configuration
+
+The `solidity` config is an optional field that can be one of the following:
+
+- The solc version to use itself. E.g. `"0.7.3"`.
+
+- An object which describes the configuration for a single compiler. It contains the following keys:
+
+  - `version`: The solc version to use. We recommend always setting this field. Default value: `"0.7.3"`.
+
+  - `optimizer`: An object with `enabled` and `runs` keys. Default value: `{ enabled: false, runs: 200 }`.
```

Comment:
This and the following field don't exist anymore. Now it has settings.

Maybe something like
```
  - `settings`:  An object with the same schema as the `settings` entry in the [Input JSON](https://solidity.readthedocs.io/en/v0.7.2/using-the-compiler.html#input-description).
```